tutorial/deploy_jimm_microk8s.rst

```diff
@@ -0,0 +1,289 @@
+    echo $VAULT_TOKEN > vault_token.txt
+
+We are now ready to move onto the next step.
+
+.. 
+    Configure OpenFGA Authorization Model
+    -------------------------------------
+    In this step we will configure the authorization model.
+
+    Create a file called auth_model.json with the following contents:
+
+    .. code:: yaml
+
+        model: >
+          {
+           include content from authorisation_model.yaml from somewhere.
+          }
+
+    run the action ``juju run jimm/leader create-authorization-model --params auth_model.yaml --wait 1m``
+
+    This step is intentionally left out as the auth model should not have to be created by the user if the 
+    approach in https://github.com/canonical/jimm/pull/1212 is adopted.
+    Currently a WIP.
+
+Configure JIMM
+--------------
+
+Nearing the end, we will configure JIMM. Here we will configure required config parameters with an explanation of what they do.
+
+Run the following commands:
+
+.. code:: bash
+    
+    # The UUID value is used internally to represent the JIMM controller in OpenFGA relations/tuples.
+    # Changes to the UUID value after deployment will likely result in broken permissions.
+    juju config jimm uuid=1234
+    # The address to reach JIMM, this will configure ingress and is also used for OAuth flows/redirects.
+    juju config jimm dns-name=test-jimm.localhost
+    # A private and public key for macaroon based authentication with Juju controllers.
+    juju config jimm public-key="<public-key>"
+    juju config jimm private-key="<private-key>"
+    # A final redirect URL for login, normally set to the URL of the Juju dashboard.
+    juju config jimm final-redirect-url="http://test-jimm.localhost/auth/whoami"
+
+Note that the public and private key pairs must be generated by the `go macaroon bakery repo <https://github.com/go-macaroon-bakery/macaroon-bakery>`__.
+To do this briefly run the following command, ensuring you have the `go` tool installed:
+
+.. code:: bash
+
+    go run github.com/go-macaroon-bakery/macaroon-bakery/cmd/bakery-keygen/v3@latest
+
+This should return a private and public key pair as below which can be used to configure JIMM.
+These values are only used internally between JIMM and Juju controllers.
+
+.. code:: json
+
+    {
+        "public": "<public-key>",
+        "private": "<private-key>"
+    }
+
+At this point you can run ``juju status`` and you should observe JIMM is active.  
+Navigate to `http://test-jimm.localhost/debug/info <http://test-jimm.localhost/debug/info>`__ to verify your JIMM deployment.
+
+Using Your JIMM Deployment
+--------------------------
+
+Now that you have JIMM running you can browse our additional guides to start adding controllers and workloads.
+
+- Setup your initial JIMM admin and configure permissiosn.
+- Learn how to add a new controller to JIMM.
+- Learn how to migrate models from existing controllers to JIMM.
+- Understand the difference between the Juju, JAAS and Jimmctl CLI tools.
```

Comment:
i "think" you should add a section on how to clean up after this tutorial.. other than that, perfect!